samples/csharp_dotnetcore/21.corebot-app-insights/FlightBookingRecognizer.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.AI.Luis;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class FlightBookingRecognizer : IRecognizer
+    {
+        private readonly LuisRecognizer _recognizer;
+
+        public FlightBookingRecognizer(IConfiguration configuration, IBotTelemetryClient telemetryClient)
+        {
+            var luisIsConfigured = !string.IsNullOrEmpty(configuration["LuisAppId"]) && !string.IsNullOrEmpty(configuration["LuisAPIKey"]) && !string.IsNullOrEmpty(configuration["LuisAPIHostName"]);
+            if (luisIsConfigured)
+            {
+                var luisApplication = new LuisApplication(
+                    configuration["LuisAppId"],
+                    configuration["LuisAPIKey"],
+                    "https://" + configuration["LuisAPIHostName"]);
+
+                // Set the recognizer options depending on which endpoint version you want to use.
+                // More details can be found in https://docs.microsoft.com/en-gb/azure/cognitive-services/luis/luis-migration-api-v3
+                var recognizerOptions = new LuisRecognizerOptionsV3(luisApplication)
+                {
+                    TelemetryClient = telemetryClient,
+                };
```

Comment:
@WashingtonKayaker Just a note to say that I merged the latest changes in from master, which included an update for the LUIS recognizer here.

Reply:
Thank you, I've updated the [docs](https://github.com/MicrosoftDocs/bot-docs-pr/pull/1909) to reflect this change.